### PR TITLE
feat(DscJobs): added DscJobs Presence

### PR DESCRIPTION
**Pull request by DiscordAnaxes**
This pr adds the DscJobs Presence.

<details>
<summary> Proof to proving the creation/modification working </summary>
<br>

Discord_FJb51lJeEs

Discord_edOYRjgyg1

</details>

### CODE REVIEW

File: websites/D/DscJobs/presence.ts [hunk 1]
```diff
@@ -0,0 +1,131 @@
+const presence = new Presence({
+    clientId: "846309777508007946"
+  }),
+  browsingStamp = Math.floor(Date.now() / 1000),
+  [page] = document.location.href.split("page=")[1].split("&"),
+  [staffPage] = document.location.href.split("page=")[1],
+  filters = document.location.href.includes("&"),
+  [search] = document.location.href.split("term=")[1].split("&");
```
Comment (ImRodry) [inline websites/D/DscJobs/presence.ts:8]: Allow me to introduce you to URLSearchParams

File: websites/D/DscJobs/presence.ts [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+    };
+
+  if (document.location.pathname === "/")
+    presenceData.details = "Viewing home page";
+  else if (document.location.pathname.includes("/search")) {
+    presenceData.details = `🔍 Searching for: ${search || "Nothing"}`;
+    presenceData.state = `📖 Results Page ${page}`;
+  } else if (document.location.pathname === "/moderators") {
+    presenceData.details = "Viewing 🔨 hireable moderators";
+    presenceData.state = `${filters ? "💿 Filters: True" : `📖 Page ${page}`}`;
```
Comment (ImRodry) [inline websites/D/DscJobs/presence.ts:27]: You shouldn’t put code terms in strings
Also remove the emojis

File: websites/D/DscJobs/presence.ts [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+    presenceData.details = `Viewing 📖 ${document
+      .querySelector("h2.cursor")
+      ?.getAttribute("data-title")} resume`;
+    presenceData.state = `❤️ ${document
+      .getElementById("likes_amount")
+      ?.getAttribute("data-title")} & 👀 ${document
+      .getElementById("views_amount")
+      ?.getAttribute("data-title")}`;
+    if (showButtons) {
+      if (showCvButton) {
```
Comment (ImRodry) [inline websites/D/DscJobs/presence.ts:86]: Combine this if

**Review by ImRodry — CHANGES_REQUESTED** [review]

File: websites/D/DscJobs/presence.ts (hunk 1 above)
Comment (EncryptedDev) [reply websites/D/DscJobs/presence.ts:8]: This please

**Review by EncryptedDev — CHANGES_REQUESTED** [review]

**Comment by EncryptedDev** [comment]
Inactive

**Pull request closed without merging**